<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminOnlyConsole</class>
 <widget class="QDialog" name="AdminOnlyConsole">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>791</width>
     <height>561</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Add Ballpark</string>
    </attribute>
    <widget class="QPushButton" name="addBallparkButton">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>180</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(191, 202, 255);</string>
     </property>
     <property name="text">
      <string>Add Ballpark</string>
     </property>
    </widget>
    <widget class="QComboBox" name="newStadiumBox">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>110</y>
       <width>131</width>
       <height>32</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>110</y>
       <width>59</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Stadiums:</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Add Souvenir</string>
    </attribute>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>90</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Souvenir Name:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="addSouvName">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>90</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>130</y>
       <width>59</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Price:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="addSouvPrice">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>130</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="newSouvStadBox">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>170</y>
       <width>131</width>
       <height>32</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>180</y>
       <width>59</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Stadium:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addSouvButton">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>230</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(200, 255, 204);</string>
     </property>
     <property name="text">
      <string>Add Souvenir</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Delete Souvenir</string>
    </attribute>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>100</y>
       <width>59</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Stadium:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>150</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Souvenir Name:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="deleteSouvStadList">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>90</y>
       <width>131</width>
       <height>32</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="deleteButton">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>220</y>
       <width>131</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(254, 166, 170);</string>
     </property>
     <property name="text">
      <string>Delete Souvenir</string>
     </property>
    </widget>
    <widget class="QComboBox" name="souvNameListDelete">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>140</y>
       <width>131</width>
       <height>32</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Edit Souvenir</string>
    </attribute>
    <widget class="QComboBox" name="editSouvNameList">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>110</y>
       <width>131</width>
       <height>32</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="editSouvButton">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>240</y>
       <width>131</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(254, 255, 179);</string>
     </property>
     <property name="text">
      <string>Edit Souvenir</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>120</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Souvenir Name:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="editSouvStadList">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>60</y>
       <width>131</width>
       <height>32</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>70</y>
       <width>59</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Stadium:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>170</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>New Price:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="newPriceInput">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>170</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>Souvenir View</string>
    </attribute>
    <widget class="QTableView" name="souvenirTableView">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>10</y>
       <width>451</width>
       <height>511</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>301</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 18pt &quot;.SF NS Text&quot;;</string>
     </property>
     <property name="text">
      <string>Souvenir View (For Testing Purposes)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Enter Stadium Name:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>170</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(230, 255, 227);</string>
     </property>
     <property name="text">
      <string>View</string>
     </property>
    </widget>
    <widget class="QComboBox" name="souvViewComboBox">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>100</y>
       <width>151</width>
       <height>32</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
